src/routes/index.js

```diff
@@ -0,0 +1,19 @@
+import messages from '../utils/messages';
+import response from '../utils/response';
+
+const routes = (router) => {
+  router
+    .route('/')
+  /**
+   * @swagger
+   * /:
+   *   get:
+   *     description: Welcome message
```

Comment:
add more documentation on how the response format looks like

Reply:
Done. Please check now